crates/astria-cli/src/sequencer/account.rs

```diff
@@ -40,19 +40,24 @@ enum SubCommand {
 }
 
 #[derive(Debug, clap::Args)]
-struct Create;
+struct Create {
+    #[command(flatten)]
```

Comment:
Flattening is not necessary for this type.

It was done for the other arg-types in this module because they use the same arguments (to avoid code duplication).

Remove `ArgsCrate` and fold them into this type.